docs/configuration/config-file-reference.md

```diff
@@ -3385,6 +3386,24 @@ query_rejection:
 # CLI flag: -compactor.tenant-shard-size
 [compactor_tenant_shard_size: <int> | default = 0]
 
+# Index size limit in bytes for each compaction partition. 0 means no limit
+# CLI flag: -compactor.partition-index-size-limit-in-bytes
+[compactor_partition_index_size_limit_in_bytes: <int> | default = 0]
+
+# Time series count limit for each compaction partition. 0 means no limit
+# CLI flag: -compactor.partition-series-count-limit
+[compactor_partition_series_count_limit: <int> | default = 0]
+
+# Index size limit in bytes for each level 1 compaction partition. 0 means no
+# limit
+# CLI flag: -compactor.partition-level1-index-size-limit-in-bytes
+[compactor_partition_level1_index_size_limit_in_bytes: <int> | default = 0]
+
+# Time series count limit for each level 1 compaction partition. 0 means no
+# limit
+# CLI flag: -compactor.partition-level1-series-count-limit
+[compactor_partition_level1_series_count_limit: <int> | default = 0]
```

Comment:
What about dividing total series count by RF in L1? So that we can get rid of this flag.
If we are concerned about late compaction, we can only divide by RF for those L1 blocks, not compacted L2 blocks? 
WDYT

Reply:
Sounds good. So use ingester ring replication_factor config, right?